core/micro-services/workflow-core/build.sbt

```diff
@@ -105,9 +105,11 @@ libraryDependencies ++= Seq(
 /////////////////////////////////////////////////////////////////////////////
 
 libraryDependencies ++= Seq(
-  "com.github.sisyphsu" % "dateparser" % "1.0.11",               // DateParser
-  "com.google.guava" % "guava" % "31.1-jre",                     // Guava
-  "org.ehcache" % "sizeof" % "0.4.3",                            // Ehcache SizeOf
-  "org.jgrapht" % "jgrapht-core" % "1.4.0",                      // JGraphT Core
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5"      // Scala Logging
+  "com.github.sisyphsu" % "dateparser" % "1.0.11",                  // DateParser
+  "com.google.guava" % "guava" % "31.1-jre",                        // Guava
+  "org.ehcache" % "sizeof" % "0.4.3",                               // Ehcache SizeOf
+  "org.jgrapht" % "jgrapht-core" % "1.4.0",                         // JGraphT Core
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",        // Scala Logging
+  "jakarta.ws.rs" % "jakarta.ws.rs-api" % "3.1.0",                  // jakarta for web errors
```

Comment:
should be removed. if not in this PR, add a TODO.

Reply:
removed